test/system/test_tpm2_createpolicy.sh

```diff
@@ -35,23 +35,21 @@
 
 declare -A expected_policy_digest=(["sha1"]="f28230c080bbe417141199e36d18978228d8948fc10a6a24921b9eba6bb1d988"
                                    ["sha256"]="33e36e786c878632494217c3f490e74ca0a3a122a8a4f3c5302500df3b32b3b8")
-declare -A hash_name=([0x4]="sha1" [0xb]="sha256")
 pcr_index=0
 
-for halg in 0x4 0xb
+for halg in sha1 sha256
 do
```

Comment:
In some tests I see that you use a mix of nice name and hex values and in other tests you convert all of them to nice names. I prefer the former to make sure that the tests exercise both options.

But I think there should be some consistency, either the tests should exercise both or all should be converted to nice names.